library/pom.xml

```diff
@@ -19,6 +19,11 @@
             <artifactId>spring-boot-starter-amqp</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
I thought you wanted flux everywhere. If it's affordable it's better not to mix flux and web. We can talk about it in dm